opencti-platform/opencti-front/src/private/components/settings/decay/DecayRuleEdition.tsx

```diff
@@ -0,0 +1,313 @@
+import React, { FunctionComponent } from 'react';
+import { graphql, useMutation } from 'react-relay';
+import { Field, FieldArray, Form, Formik } from 'formik';
+import Drawer, { DrawerVariant } from '@components/common/drawer/Drawer';
+import * as Yup from 'yup';
+import * as R from 'ramda';
+import makeStyles from '@mui/styles/makeStyles';
+import { AddOutlined, Delete } from '@mui/icons-material';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import Tooltip from '@mui/material/Tooltip';
+import Box from '@mui/material/Box';
+import { InformationOutline } from 'mdi-material-ui';
+import ListItemText from '@mui/material/ListItemText';
+import { FormikConfig } from 'formik/dist/types';
+import AutocompleteField from '../../../../components/AutocompleteField';
+import { useFormatter } from '../../../../components/i18n';
+import ItemIcon from '../../../../components/ItemIcon';
+import MarkdownField from '../../../../components/MarkdownField';
+import TextField from '../../../../components/TextField';
+import type { Theme } from '../../../../components/Theme';
+import SwitchField from '../../../../components/SwitchField';
+import { fieldSpacingContainerStyle } from '../../../../utils/field';
+import useAuth from '../../../../utils/hooks/useAuth';
+import { handleError } from '../../../../relay/environment';
+import { DecayRule_decayRule$data } from './__generated__/DecayRule_decayRule.graphql';
+
+const useStyles = makeStyles<Theme>((theme) => ({
```

Comment:
`makeStyles` is deprecated. Try to put style directly in JSX using `style={{...}}` for html elements or `sx={{...}}` for MUI elements